.github/workflows/actions.yaml

```diff
@@ -14,12 +14,11 @@ jobs:
         with:
           matrix: |
             python-version {2.7}, tox-env {py27-django111}
-            python-version {3.5}, tox-env {py35-django111,py35-django22}
-            python-version {3.6}, tox-env {py36-django111,py36-django22,py36-django30,py36-django31,py36-django32}
```

Comment:
Believe it or not, I know of some deployments using some of these combinations with this module, so I would rather not remove them quite yet.

Reply:
If you want I can make a change to ditch the system Python that's built into Github Actions and run the tests in Docker instead where we can trivially pick any Python version. Let me know if that's desirable (but I reckon I'd open a separate PR for this).